### ISSUE TITLE
Ground joints disappear when changing units. Also disappears when 'Update Object Scale' button is clicked

### ISSUE THREAD

**Issue by AlexG1031** [PMKS-Web/PMKSWeb#203]

**Comment by KohmeiK** [comment]
Yeah, I've seen this before. I think when the absolute size of the SVG image becomes too small, the browser tries to optimize but not showing it. I think the solution is to scale everything up by 100 or 1000 so that we have to zoom a significantly for this to happen again. This should be a purely visual bug though so not critical.
